src/js/components/icons/status/Disabled.js

```diff
@@ -1,22 +1,26 @@
 // (C) Copyright 2014-2015 Hewlett-Packard Development Company, L.P.
 
 var React = require('react');
+var IntlMixin = require('../../../mixins/GrommetIntlMixin');
 
 var Disabled = React.createClass({
 
+  mixins: [IntlMixin],
+
   render: function() {
     var className = 'status-icon status-icon-disabled';
     if (this.props.className) {
       className += ' ' + this.props.className;
     }
     return (
-      <svg className={className} viewBox="0 0 24 24" version="1.1">
-        <g className={"status-icon__base"}>
-          <rect x="0" y="0" width="24" height="24" stroke="none"></rect>
+      <svg className={className} viewBox="0 0 24 24" role="img" aria-labelledby="title-icon-disabled" version="1.1">
+        <title id="title-icon-disabled">{this.getGrommetIntlMessage('Disabled')}</title>
+	<g className={"status-icon__base"}>
+          <rect role="presentation" x="0" y="0" width="24" height="24" stroke="none"></rect>
```

Comment:
Attribute `role="img"` is on the `svg` element.  `role="presentation" is on the drawing elements so screen readers do not read the title more than once.  Example: If you use a screen reader like [NVDA](http://www.nvaccess.org/) and  do not put the `role="presentation"` on both the `circle` and `path` element NVDA will read 'graphic OK OK' after adding `role="presentation" on those element NVDA reads 'graphic OK'. This output is correct and not confusing to the sreen reader user.